wisdem/glue_code/glue_code.py

```diff
@@ -37,10 +37,10 @@ def setup(self):
         if modeling_options['flags']['blade']:
             self.add_subsystem('ccblade',   CCBladeTwist(modeling_options = modeling_options, opt_options = opt_options)) # Run standalong CCBlade and possibly determine optimal twist from user-defined margin to stall
             self.add_subsystem('wt_class',  TurbineClass())
-            self.add_subsystem('elastic',   RotorElasticity(modeling_options = modeling_options, opt_options = opt_options))
-            self.add_subsystem('sse',       ServoSE(modeling_options = modeling_options)) # Aero analysis
+            self.add_subsystem('re',        RotorElasticity(modeling_options = modeling_options, opt_options = opt_options))
```

Comment:
What do you think about naming all these groups more verbosely, for example `re` -> `rotor_elasticity`? It'd be worthwhile to make them more clear to new users, in my opinion

Reply:
sounds like a good idea. I will re-rename them tomorrow